app/src/main/res/layout-land/activity_save_book.xml

```diff
@@ -64,7 +62,33 @@
             android:inputType="text"
             app:layout_constraintEnd_toStartOf="@+id/guidelineRight"
             app:layout_constraintStart_toEndOf="@+id/guidelineLeft"
-            app:layout_constraintTop_toBottomOf="@+id/topBarrier"/>
+            app:layout_constraintTop_toBottomOf="@+id/topBarrier" />
+
+        <EditText
+            android:id="@+id/yearEditText"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_normal"
+            android:background="@android:color/transparent"
+            android:hint="Year"
```

Comment:
All text presented to the user should be defined in _strings.xml_ because it allows internationalization.